src/app/projects/project-details/project-details.component.html

```diff
@@ -29,29 +29,30 @@ <h4>{{project.name}}</h4>
       </div>
     </div>
     <div class="project-page-body">
-      <wh-tabview-x *ngIf="project.buildHistory">
+      <wh-tabview-x *ngIf="project.buildHistory" [(activeIndex)]="activeTabIndex" (activeIndexChange)="tabviewTrackerService.updateQueryParamsWithTabIndex($event)">
```

Comment:
The `activeIndex` field is only used to set its initial value. Could something like this be done instead?

```suggestion
      <wh-tabview-x *ngIf="project.buildHistory" activeIndex="{{tabviewTrackerService.getTabIndexFromQueryParams()}}" (activeIndexChange)="tabviewTrackerService.updateQueryParamsWithTabIndex($event)">
```